m3-sys/cminstall/src/config/I386_DJGPP

```diff
@@ -36,6 +36,7 @@ M3_BACKEND_MODE = "C"
 include ("cm3cfg.common")
 
 proc compile_c(source, object, options, optimize, debug) is
+  options       = escape(subst_chars(options, "\\", "/"))
```

Comment:
Are you sure you need escape?

Reply:
And this "escape" is needed escape.See description of patch